tools/webhook-ingestion-pipeline/webhook_endpoint/pubsub_publisher.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+from google.cloud.pubsub_v1 import PublisherClient
+
+
+class PubSubPublisher:
+
+    def __init__(self, project_id):
+        self.project_id = project_id
+        self.client = PublisherClient(
+            client_config = {
```

Comment:
nit: define this as a "private" static class member dict constant as it never changes.
Also clean up the formatting.